rag_experiment_accelerator/config/config.py

```diff
@@ -145,8 +148,10 @@ def __init__(
         self.MAIN_PROMPT_INSTRUCTION = (
             config_json["main_prompt_instruction"]
             if "main_prompt_instruction" in config_json
-            else main_prompt_instruction
+            else "main_instruction_short.txt"
```

Comment:
Rather than hard-code this string here, shouldn't this use `main_instruction_short`?
If not shouldn't the default be defined as a const somewhere rather than hardcoded here?